Don't use IUCLC flag if it is not supported.
Upstream commit:d466ea1

--- codels/imuRequestTaskCodels.c~	2008-07-16 16:54:40.000000000 +0200
+++ codels/imuRequestTaskCodels.c	2014-08-19 15:44:46.946220757 +0200
@@ -300,7 +300,11 @@
    cfsetispeed(&t, imuData->baud);
    t.c_cflag &= ~(CSIZE|PARENB);
    t.c_cflag |= CS8 | CLOCAL;
-   t.c_iflag &= ~(ISTRIP|ICRNL|INLCR|IGNCR|IXON|IXOFF|IUCLC);
+   t.c_iflag &= ~(ISTRIP|ICRNL|INLCR|IGNCR|IXON|IXOFF
+#ifdef IUCLC
+                  |IUCLC
+#endif
+     );
    t.c_oflag &= ~OPOST;
    t.c_lflag &= ~(ICANON|ISIG|IEXTEN|ECHO);
    t.c_cc[VMIN] = 0;
